ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarGroupViewDemoController.swift

```diff
@@ -66,15 +104,20 @@ class AvatarGroupViewDemoController: DemoController {
         settingsTitle.text = "Settings"
         container.addArrangedSubview(settingsTitle)
 
-        addRow(text: "Avatar count", items: [incrementBadgeButton, decrementBadgeButton], textStyle: .footnote, textWidth: Constants.settingsTextWidth)
+        let avatarCountButtonsStackView = UIStackView(arrangedSubviews: [incrementBadgeButton.view, decrementBadgeButton.view])
```

Comment:
Any specific reason for not using container of `DemoController`?

Reply:
I am still in favor of `UIStackview` extension doesn't have to be part of this PR :)

Reply:
I added the UIStackView to have control over the size of the buttons and the space between them.
Just passing them on the addRow was changing the buttons size in a way that didn't look good.